File: Fundamentals of C programming/step07/7.2/7.2.01.c
/*

Последовательность
Вывести последовательность чисел, поступившую на вход.

Входные данные:
На вход программы поступает последовательность целых чисел. Количество чисел в последовательности заранее неизвестно. Но известно, что в конце последовательности записано число -9999 и в последовательности всегда есть хотя бы одно число, кроме -9999.

Выходные данные:
Вывести элементы последовательности на экран через пробел.

Sample Input:

929 440 -77 673 44 -947 -475 -20 919 310 -581 -843 834 532 589 452 -578 -899 11 26 824 -952 -264 254 148 156 -528 280 -9999

Sample Output:

929 440 -77 673 44 -947 -475 -20 919 310 -581 -843 834 532 589 452 -578 -899 11 26 824 -952 -264 254 148 156 -528 280 -9999

*/

#include <stdio.h>

int main(void) {

  int n;

  while(n!=-9999){
	scanf("%d",&n);
	printf("%d ",n);
	}
  
  return 0;
}

File: Diving into JavaScript: For Beginners/step07/7.02/7.2.01.js
/*

Попросите пользователя ввести строку. Если длина строки меньше 5 символов, добавьте к строке "!" и выведите результат, 
иначе добавьте к строке "?" и также выведите результат.

Sample Input 1:

Братва

Sample Output 1:

Братва?

Sample Input 2:

Брат

Sample Output 2:

Брат!

*/

let str = prompt();

if (str.length < 5) {
    console.log(str+'!');
} else {
    console.log(str+'?');
}
